vuln-reach/src/javascript/lang/symbol_table.rs

```diff
@@ -265,7 +274,9 @@ impl<'a> SymbolTable<'a> {
             // If formal_parameters nodes has a statement_block next sibling, it is a
             // regular function, otherwise it is an arrow function and the
             // identifier doesn't belong to a scope.
-            let body = parent.parent().unwrap().child_by_field_name("body").unwrap();
+
+            let grandparent = cursor.clone().goto_parent().unwrap();
```

Comment:
Is this the correct way to use the new `Cursor` API? This will not change the original cursor, right?